client/v3/maintenance.go

```diff
@@ -181,13 +181,16 @@ func (m *maintenance) AlarmDisarm(ctx context.Context, am *AlarmMember) (*AlarmR
 			return nil, toErr(ctx, err)
 		}
 		ret := AlarmResponse{}
+		header := ar.Header
 		for _, am := range ar.Alarms {
 			dresp, derr := m.AlarmDisarm(ctx, (*AlarmMember)(am))
 			if derr != nil {
 				return nil, toErr(ctx, derr)
 			}
 			ret.Alarms = append(ret.Alarms, dresp.Alarms...)
+			header = dresp.Header
 		}
+		ret.Header = header
```

Comment:
```suggestion
		ret.Header = ar.Header
		for _, am := range ar.Alarms {
			dresp, derr := m.AlarmDisarm(ctx, (*AlarmMember)(am))
			if derr != nil {
				return nil, toErr(ctx, derr)
			}
			ret.Alarms = append(ret.Alarms, dresp.Alarms...)
			ret.Header = dresp.Header
		}
```

Reply:
This looks like a reasonable fix for a bug, however I would want to make sure that it's properly reviewed. Can you send it in separate PR?